### ISSUE TITLE
X11R4

### ISSUE THREAD

**Issue by gattilorenz** [retrohun/blog#3]
I have X11R4 sources + patches for Xenix, but I never managed to get it to compile myself.

If you want to give it a shot, let me know.

**Comment by retrohun** [comment]
> I have X11R4 sources + patches for Xenix, but I never managed to get it to compile myself.
> 
> If you want to give it a shot, let me know.

Wow, is it freely downloadable or restricted software? archive.org, vogons or other similar forum members would be ideal targets!
Guessing but it is only for Xenix386+?

**Comment by gattilorenz** [comment]
Freely downloadable. X11R4 is, of course, free. X386 as well. The patches for Xenix were on Usenet.

I've uploaded the archive here: https://archive.org/details/x-11-r-4-x-386.7z

The annoying thing is putting everything in a VM. Now that networking works in qemu, I guess it is possible to just use ftp or samba (yes, samba! There's a binary at ftp://ftp.celestial.com/pub/sco-ports/xnx and it works: https://tinyurl.com/SambaXenixScreenshots ). A few years ago I had to do it with floppy images...
